src/lib.rs

```diff
@@ -49,13 +49,20 @@
 
 extern crate static_assertions;
 
+#[cfg(feature = "c64")]
 pub mod c64;
+#[cfg(feature = "c64")]
 pub mod cia;
+#[cfg(feature = "cx16")]
 pub mod cx16;
+#[cfg(feature = "mega65")]
```

Comment:
Here the `mega65` feature should also include `sid` and `vic2`.

Reply:
Would it make sense to also add features for the indiviual components (e.g. `sid` and `vic2`) in case someone wants to use it on a platform for which no dedicated feature exists?

Reply:
Yes, that might be useful. Is it also possible to filter the `examples/`? For example, `mega65-*.rs` will not compile unless the mega65 feature is enabled.